integration_tests/broker_suite_test.go

```diff
@@ -44,8 +45,7 @@ func TestSuite(t *testing.T) {
 		logger := lager.NewLogger("compose-broker")
 		logger.RegisterSink(lager.NewWriterSink(os.Stdout, config.LogLevel))
 
-		catalog := catalog.New()
-		catalogData := `
+		catalogData := bytes.NewBuffer([]byte(`
```

Comment:
Instead of having to cast this to a `[]byte`, you could use a `strings.Reader` here.